eclipse-collections/src/main/java/org/eclipse/collections/impl/list/primitive/IntInterval.java

```diff
@@ -1007,4 +1016,82 @@ public int next()
             throw new NoSuchElementException();
         }
     }
+
+    private static final class IntIntervalSpliterator implements Spliterator.OfInt
+    {
+        private int current;
+        private final int to;
+        private final int step;
+        private final boolean isAscending;
+
+        private IntIntervalSpliterator(int from, int to, int step)
+        {
+            this.to = to;
+            this.step = step;
+            this.current = from;
```

Comment:
Please initialize the fields in the same order they are declared

Reply:
@david-gang this review comment is still pending to be addressed.